fnl_doc/tutorial/00_basics.fnl

```diff
@@ -0,0 +1,172 @@
+($
+((h 1) "Basic tutorial")
+
+(p
+"
+Start by opening an interactive shell. First install git and python 3.
+Then install gurklang and open the interactive shell by entering the following into the command line
+")
+(gurklang
+"""
+$ git clone https://github.com/gurkult/py-gurklang
+$ cd py-gurklang
+$ python -m pip install -r requirements.txt
+$ python -m gurklang
+"""
+)
+(md "on some platforms, `python` will be `python3` or `py`")
+(p "you should see something like")
+(gurklang
+"""
+Gurklang v0.0.1
+--------------
+>>>
+""")
+(p "now you can enter commands for the gurklang runtime.")
+((h 2) "The Value Stack")
+
+(md
+"
+Gurklang uses a stack to store intermediate results.
+You can think of a stack as a column of crates, with each new value going on top and all operations working from top to bottom. You start with no 'crates'. Lets add the first value (crate) to the stack.
+enter `55` into the interactive shell
+")
+(gurklang
+"""
+>>> 55
+>>>
+""")
+(p
+"
+it may seem like nothing happened, but this added the first value to the stack. The integer 55.
+entering peek shows the current values on the stack. Lets put more numbers on and try it out
+"
+(gurklang
+"""
+>>> 44
+>>> 11
+>>> 0
+>>> -9
+>>> peek
+(-9 (0 (11 (44 (55 ())))))
+""")
+(md "
+we can see that we have the numbers on the stack.
+The topmost value of the stack is in the outermost parentheses. As we added -9 last, it is on top.
+Lets say we want to remove the top value. For that we use the function `drop`.
```

Comment:
Should we use the word `function`? Maybe explain briefly what it means? Or maybe not name it at all? Something like:
"for that we use `drop`".